observability/interceptor.go

```diff
@@ -84,10 +85,10 @@ func (i *Interceptor) recordVerbCallMetrics(ctx context.Context, verb string, st
 
 	attributes := []attribute.KeyValue{
 		attribute.String(verbRefKey, verb),
-		attribute.String(sourceVerbKey, sourceVerbRef.Name),
-		attribute.String(sourceModuleKey, sourceVerbRef.Module),
-		attribute.String(destVerbKey, caller.Value().Name),
-		attribute.String(destModuleKey, caller.Value().Module),
+		attribute.String(SourceVerbKey, sourceVerbRef.Name),
+		attribute.String(SourceModuleKey, sourceVerbRef.Module),
+		attribute.String(DestVerbKey, caller.Value().Name),
```

Comment:
`.Value()` can panic - you should almost never use this, but instead check that the value is valid before using it.